variables.tf

```diff
@@ -15,6 +15,14 @@ variable "cert_read_role_accounts_allowed" {
   default     = []
 }
 
+variable "client_dns_search_domain" {
+  description = "The DNS search domain to be pushed to VPN clients."
```

Comment:
Can these new variables be specified as type `string`?

Reply:
dun: 8e78b64